packages/apps/human-app/server/src/common/config/app-config.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { HttpModule } from '@nestjs/axios';
+import { GatewayConfigService } from './gateway-config.service';
+import { EnvironmentConfigService } from './environment-config.service';
+
+@Module({
+  imports: [HttpModule],
```

Comment:
I don't see HttpModule being used in either GatewayConfigService or EnvironmentConfigService.